examples/core/core_input_gamepad.c

```diff
@@ -150,24 +174,24 @@ int main(void)
                     // Draw axis: left joystick
                     Color leftGamepadColor = BLACK;
                     if (IsGamepadButtonDown(gamepad, GAMEPAD_BUTTON_LEFT_THUMB)) leftGamepadColor = RED;
-                    DrawCircle(319, 255, 35, leftGamepadColor);
+                    DrawCircle(319, 255, 35, BLACK);
```

Comment:
I don't think this was meant to be hardcoded to `BLACK`. It's conditionally red to "light up" and visually show the user they pressed it, right?

Reply:
No, this change is correct.
All other outer circles are black: [L101](https://github.com/raysan5/raylib/blob/7a4a84a5619754b8170737726a14b368e8cbc5bf/examples/core/core_input_gamepad.c#L101), [L109](https://github.com/raysan5/raylib/blob/7a4a84a5619754b8170737726a14b368e8cbc5bf/examples/core/core_input_gamepad.c#L109), [L161](https://github.com/raysan5/raylib/blob/7a4a84a5619754b8170737726a14b368e8cbc5bf/examples/core/core_input_gamepad.c#L161).
Only the inner circles get lit red when active: [L104](https://github.com/raysan5/raylib/blob/7a4a84a5619754b8170737726a14b368e8cbc5bf/examples/core/core_input_gamepad.c#L104), [L112](https://github.com/raysan5/raylib/blob/7a4a84a5619754b8170737726a14b368e8cbc5bf/examples/core/core_input_gamepad.c#L112), [L156](https://github.com/raysan5/raylib/blob/7a4a84a5619754b8170737726a14b368e8cbc5bf/examples/core/core_input_gamepad.c#L156), [L164](https://github.com/raysan5/raylib/blob/7a4a84a5619754b8170737726a14b368e8cbc5bf/examples/core/core_input_gamepad.c#L164).